flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "end-of-life": "This application is no longer maintained."
```

Comment:
eol-rebase, needs to wait for the new submission.

> We're planning a new release, which will have a new App ID that will supercede the current one, at which point we'll update the manifest with the appropriate end-of-life-rebase information.

Reply:
He decided "Vocal is End-of-Life"

https://nathandyer.me/2024/11/26/end-of-vocal.html